translations/handleDesktopTranslations.sh

```diff
@@ -12,16 +12,58 @@ gpg --list-keys
 git clone git@github.com:nextcloud/desktop.git
 cd desktop
 
-lupdate-qt5 src/gui/ src/cmd/ src/common/ src/crashreporter/ src/csync/ src/libsync/ -ts translations/client_en.ts
+# Generate source translation files for master and stable branches
+mkdir /branches
+
+versions="stable-2.6 master"
+if [[ -f '.tx/backport' ]]; then
+  versions="$(cat .tx/backport) master"
```

Comment:
what is backport? Seems not to be used anymore?

Reply:
This is from #165 that would allow the repo to specify their own list of stable branches, but @tobiasKaminsky is right, there should be just one approach ;)

Reply:
See my comment above ;)

Reply:
Should now be clarified :)